doc/platforms/openmote.md

```diff
@@ -56,34 +64,42 @@ Instructions to flash for different OS are given below.
     * Nodes can also be programmed via the serial boot loader in the cc2538. In `tools/cc2538-bsl/` you can find `cc2538-bsl.py` python script, which can download firmware to your node via a serial connection. If you use this option you just need to make sure you have a working version of python installed. You can read the README in the same directory for more info.
 
 * On Linux:
-    * Nodes can be programmed with TI's [UniFlash] tool. With UniFlash, use the file with `.elf` extension.
-    * Nodes can also be programmed via the serial boot loader in the cc2538. No extra software needs to be installed.
+    * Nodes can be programmed with TI's UniFlash tool. With UniFlash, use the file with `.elf` extension.
```

Comment:
Is there an .elf extension output from the build? Is it not a .openmote or something like that?